Marlin/Marlin_main.cpp

```diff
@@ -2412,7 +2412,7 @@ static void clean_up_after_endstop_or_probe_move() {
 
     const float nx = rx - (X_PROBE_OFFSET_FROM_EXTRUDER), ny = ry - (Y_PROBE_OFFSET_FROM_EXTRUDER);
 
-    if (printable
+    if (!printable
```

Comment:
Strange nobody with an eccentric probe noticed this before: with G33 probe (printable = false) check if nozzle is reachable : default more restrictive probe is reachable check !!!

Reply:
Argh, yeah that was me. I must have been doing too much crack in August. https://github.com/MarlinFirmware/Marlin/commit/0938c62b4826e62549f0f86818fd8d4c8e94d228

Thanks for this correction!